src/test/java/com/fasterxml/jackson/databind/ser/JsonAssert.java

```diff
@@ -0,0 +1,79 @@
+package com.fasterxml.jackson.databind.ser;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+
+public class JsonAssert {
```

Comment:
I don't like name here; JsonAssert sounds like a class from unit test framework.
Class could use Javadoc to explain the intent.